testbook/execute.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import nbformat
+from testbook.testbook.client import TestbookNotebookClient
+
+
+def execute_cell(notebook_filename, cell_index):
+    def decorator(func):
```

Comment:
We may want to use `@functools.wraps(func)` https://stackoverflow.com/questions/147816/preserving-signatures-of-decorated-functions to preserve function signatures.